docs/debugging/README.md

```diff
@@ -0,0 +1,13 @@
+# Debugging
+
+## Providers
+
+See [Debugging Pulumi Provider Packages](https://www.pulumi.com/docs/iac/packages-and-automation/pulumi-packages/debugging-provider-packages/).
+
+:::{toctree}
+:maxdepth: 1
+:titlesonly:
+
+/docs/debugging/engine
+/docs/debugging/providers_and_languages
```

Comment:
Let's stick with the convention of `kebab-case` for docs files/links if possible.